include/deal.II/base/time_stepping.h

```diff
@@ -373,6 +399,120 @@ namespace TimeStepping
 
 
 
+  /**
+   * LowStorageRungeKutta is derived from RungeKutta and implement the explicit
+   * methods.
```

Comment:
Update comment. Can you add a sentence for motivation why one would want to use these?